nix/tullia.nix

```
@@ -104,7 +104,7 @@ rec {
         #lib.io.github_push
         #input: "${ciInputName}"
         #repo: "${repository}"
-        #branch: "master|staging|trying"
+        #branch: "master|release/1.35"
```

This is a regex, you can match all releases:
```suggestion
        #branch: "master|release/.*"
```